### ISSUE TITLE
Incorrect preview color of the custom key signature in `Invert score` mode in contrast theme

### ISSUE THREAD

**Issue by PtJade-Ceramic** [musescore/MuseScore#28424]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. [Open a score with this software](https://musescore.org/en/node/329673#Opening_a_score);
2. Change the [theme and color](https://musescore.org/en/node/332084#Theme_and_Colors) to the following:
   1. ✅ Select `Black`
   2. ✅ Tick `Enable high contrast`
   3. ✅ Tick `Invert score`
3. Try [creating a custom key signature](https://musescore.org/en/node/329687#custom-key-sig).

| Element | Actual color | Expected color |
| --- | --- | --- |
| Paper | white | black |
| Staff lines | black | white |
| Clef | white | white, same as the color of the paper, hard to see |

### Supporting files, videos and screenshots

![Image](https://github.com/user-attachments/assets/63a821c1-3993-4ab0-bc25-53ccfecf41f3)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.5.2-251141402, revision: github-musescore-musescore-ac9d3bc

### Regression

I was unable to check

### Operating system

版本	Windows 11 家庭版 版本号	24H2 操作系统版本	26100.2033 体验	Windows Feature Experience Pack 1000.26100.23.0

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
It turns out this is a duplicate of https://github.com/musescore/MuseScore/issues/27593, but nobody knew, because that issue was already closed as fixed by the time that this one was created. Anyway, so this is fixed in 4.6!
